.github/workflows/workflow.yml

```diff
@@ -34,14 +32,18 @@ jobs:
           const branchName = context.payload.base_ref.replace('refs/heads/', '');
 
           const list = await github.rest.git.listMatchingRefs({
-            owner: 'volbobvol',
-            repo: 'action-target',
+            owner: ${{ env.TARGET_REPO_OWNER }},
+            repo: ${{ env.SUBMIT_SIGNING_REQUEST_TARGET_REPO }},
             ref: 'heads/' + branchName
           });
-          console.log(JSON.stringify(list));
+          
+          if (list.data.length === 0) {
+            console.log('Branch not found in the terget repository');
+          }
+
           await github.rest.git.createRef({
-            owner: 'volbobvol',
-            repo: 'action-target',
+            owner: ${{ env.TARGET_REPO_OWNER }},
+            repo: ${{ env.SUBMIT_SIGNING_REQUEST_TARGET_REPO }},
             ref: 'refs/tags/${{ github.ref_name }}',
             sha: list.data[0].object.sha
           })
```

Comment:
Did you verify that creating a tag for a commit hash that does not exist on the target repository fails?

Reply:
I take the latest SHA for our current target branch and create a tag there 
empty SHA doesn't work 
invalid SHA I didn't try, but what for?